tests/Browser/PilotPassportChallengeTest.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Tests\Browser;
+
+use App\PilotPassportAward;
+use App\PilotPassportEnrollment;
+use App\PilotPassportLog;
+use App\RealopsPilot;
+use Config;
+use Laravel\Dusk\Browser;
+use Tests\DuskTestCase;
+
+class PilotPassportChallengeTest extends DuskTestCase {
+
+    const TEST_AIRPORT_ID = 'GSO';
+    const TEST_CHALLENGE_ID = 1;
+
+    public function test_home_public(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->assertSee('What is the ZTL Pilot Passport Challenge?');
+        });
+    }
+
+    public function test_home_authenticated(): void {
+        $this->loginSamplePilot();
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->assertSee('Welcome, ');
+        });
+    }
+    
+    public function test_enrollments_view(): void {
+        $this->loginSamplePilot();
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->clickLink('Enrollments')
+                    ->assertSee('Select a path below and enroll to get started.');
+        });
+    }
+
+    public function test_enroll(): void {
+        $this->loginSamplePilot();
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+            ->clickLink('Enrollments')
+            ->press('@enroll_1')
+            ->assertSee('You are now enrolled in the ZTL Pilot Passport program!');
+        });
+    }
```

Comment:
This is another good spot to check the backend and make sure the pilot was actually enrolled

Reply:
Done!